### PR TITLE
Profiles: Support for running with existing profiles + saving profile after a login

### PR DESCRIPTION
**Pull request by ikreymer**

### CODE REVIEW

File: create-login-profile.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+      "-screen",
+      "0",
+      process.env.GEOMETRY,
+      "-ac",
+      "+extension",
+      "RANDR"
+    ]);
+  }
+
+  //await new Promise(resolve => setTimeout(resolve, 2000));
```
Comment (emmadickson) [inline create-login-profile.js:65]: Why is this commented out?

File: crawler.js [hunk 2]
```diff
@@ -43,6 +44,7 @@ class Crawler {
 
     this.userAgent = "";
     this.behaviorsLogDebug = false;
+    this.profileDir = fs.mkdtempSync(path.join(os.tmpdir(), "profile-"));
```
Comment (emmadickson) [inline crawler.js:47]: so will the profile information be stored in this temp folder in the docker image?

Comment (ikreymer) [reply crawler.js:47]: yeah, the profile is created in a temp directory, but then exported as a tarball and copied to the location specified by `--filename`. Need to add docs to explain this, and ideally, add a test as well..

File: create-login-profile.js (hunk 1 above)
Comment (ikreymer) [reply create-login-profile.js:65]: Was just testing something during development and thought a delay was needed there, but that turned out to not be needed, so should probably just remove this.

**Review by emmadickson — APPROVED** [review]
LGTM

**Pull request merged**